remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala

```diff
@@ -368,6 +369,8 @@ class NettyTransport(val settings: NettyTransportSettings, val system: ExtendedA
 
   private def newPipeline(channel: Channel): ChannelPipeline = {
     val pipeline = channel.pipeline()
+    pipeline.addLast("FlushConsolidationHandler",
+      new FlushConsolidationHandler(FlushConsolidationHandler.DEFAULT_EXPLICIT_FLUSH_AFTER_FLUSHES, true))
```

Comment:
should this be `addFirst`? 

From Netty javadoc:

The [FlushConsolidationHandler](https://netty.io/4.1/api/io/netty/handler/flush/FlushConsolidationHandler.html) should be put as first [ChannelHandler](https://netty.io/4.1/api/io/netty/channel/ChannelHandler.html) in the [ChannelPipeline](https://netty.io/4.1/api/io/netty/channel/ChannelPipeline.html) to have the best effect.

Reply:
The others is using addLast too, so it's the effective first. But I think I can change it to add first for more clearly ?

Reply:
@pjfanning I have change the code to make use of `addFirst`